src/app/routes/_broadcast.router.js

```diff
@@ -59,11 +84,17 @@ export default (app, router, admin) => {
   router.put('/api/broadcast/:id', admin, async (req, res) => {
     try {
       let broadcast = await Broadcast.findById(req.params.id);
+
       for (const key of Object.keys(req.body)) {
         broadcast[key] = req.body[key];
+
       }
       await broadcast.save();
 
+      if (broadcast.ended && !broadcast.videoUrl) {
+        checkBroadcastLink(broadcast);
```

Comment:
Could you please add this to the new broadcast stop script? Because when we activate the script, this manual method of stopping the broadcast isn't used anymore, I think